services-js/commissions-app/src/server/services/Email.test.ts

```diff
@@ -0,0 +1,17 @@
+import { namesFromIds } from './Email';
+
+describe('namesFromIds', () => {
```

Comment:
Good use of unit test, especially since there’s a type conversion that’s necessary between integer and string.